qiskit/synthesis/stabilizer/stabilizer_decompose.py

```diff
@@ -0,0 +1,72 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2023.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""
+Circuit synthesis for a stabilizer state preparation circuit.
+"""
+# pylint: disable=invalid-name
+
+from qiskit.circuit import QuantumCircuit
+from qiskit.exceptions import QiskitError
+from qiskit.quantum_info.states import StabilizerState
+from qiskit.synthesis.linear_phase import synth_cz_depth_line_mr
+from qiskit.synthesis.clifford.clifford_decompose_layers import (
+    synth_clifford_layers,
+    _default_cz_synth_func,
+)
+
+
+def synth_stabilizer_layers(
+    stab,
+    cz_synth_func=_default_cz_synth_func,
+    cz_func_reverse_qubits=False,
+    validate=False,
+):
```

Comment:
Yeah, the main API is good, both for ``synth_stabilizer_layers`` and for ``synth_stabilizer_depth_lnn``.